### PR TITLE
Simplify and clarify calendar contact form information

### LINKED ISSUE #1338: It is unclear how contact info on a calendar and contact info on a partner are used
## User story

As a user who can edit partner details and add calendars I need know how the contact info from the 2 forms (partner vs calendar) is used so that I understand which info to put there,

## Acceptance criteria

- [ ] Change calendar public contact info help text to "the contact info for the email account that manages this calendar"
- [ ] Remove partnership contact info

## Implementation notes & questions
image
image

## Implementation plan

_To be written by the developer_

### PR DESCRIPTION
**Pull request by ivankocienski**
Fixes #1338 

- Change of copy on the 'public contact' area
- Removal of partnership contact form area

### CODE REVIEW

**Comment by ivankocienski** [comment]
Should the contact info fields on Calendars be dropped?

**Comment by kimadactyl** [comment]
Commented on the issue

File: app/views/admin/calendars/_form.html.erb [hunk 1]
```diff
@@ -65,24 +65,14 @@
     <div class="row">
       <div class="col-md-6">
         <h4>Public Contact</h4>
```
Comment (kimadactyl) [inline app/views/admin/calendars/_form.html.erb:67]: we can remove this h4 as well

Comment (ivankocienski) [reply app/views/admin/calendars/_form.html.erb]: done

**Review by kimadactyl — APPROVED** [review]

**Pull request merged**